docker-compose.yml

```diff
@@ -81,6 +81,7 @@ services:
     volumes:
       - ./etc/delfin:/etc/delfin
       - db_data:/var/lib/delfin
+      - metrics_dir:${DELFIN_METRICS_DIR}
```

Comment:
Please add  ```DELFIN_METRICS_DIR``` related info in [installation steps](https://github.com/sodafoundation/delfin/pull/613/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R15)

Reply:
Thanks, fixed